packages/bsky/tests/db.test.ts

```diff
@@ -1,10 +1,7 @@
-import { once } from 'events'
 import { sql } from 'kysely'
 import { wait } from '@atproto/common'
 import { TestNetwork } from '@atproto/dev-env'
-import { Database } from '../src'
-import { PrimaryDatabase } from '../src/db'
-import { Leader } from '../src/db/leader'
+import { Database, PrimaryDatabase } from '../src'
 
 describe('db', () => {
```

Comment:
since this is only used for mocks, you can probably just delete this whole test 🤷‍♂️

Reply:
actually same with several of these - did-cache, duplicate-records, handle-invalidation, etc

Reply:
Sort of torn on this!  If the code is still around and is important for correctness of the other tests, maybe it's worth keeping them around in some form.  Pretty sure it will remain low cost to maintain.  Perhaps they could just be collapsed into a single file and condensed a bit.